bin/tag-release.sh

```diff
@@ -19,9 +19,25 @@ while [[ $# -ge 1 ]]; do
   shift # past argument or value
 done
 
+# compare master and stage
+echo "Comparing master branch to staging..."
+git fetch "$moz_git_remote"
+stage_hash=`git rev-parse $moz_git_remote/stage`
+master_hash=`git rev-parse master`
+if [ $stage_hash != $master_hash ]; then
```

Comment:
style NIT: always wrap variables in double quotes:

```suggestion
if [[ "$stage_hash" != "$master_hash" ]]; then
```

This protects against the case when one is blank and would throw a syntax error without the quotes.